strings.txt

```diff
@@ -18439,11 +18692,62 @@
     it = Per usare la navigazione e altri strumenti, dobbiamo accedere alla tua geolocalizzazione.
     fr = Pour utiliser la navigation et d'autres fonctionnalités, nous devons pouvoir accéder à votre position géographique.
     ja = ナビゲーションやその他の機能の使用のために、アプリがGeolocationにアクセスする必要があります
+    vi = Để sử dụng tính năng dẫn đường và các tính năng khác, chúng tôi cần truy cập vị trí địa lý của bạn.
 
   [dialog_incorrect_feature_position]
-    en = Change the location
+    en = Change location
     ru = Измените местоположение
+    tr = Konumu değiştir
+    sv = Ändra placering
+    nb = Endre plassering
+    ko = 위치 변경
+    id = Ubah lokasi
+    hu = Helyszín megváltoztatása
+    de = Standort wechseln
+    fi = Vaihda sijaintia
+    nl = Locatie wijzigen
+    da = Skift lokation
+    cs = Změnit umístění
+    zh-Hans = 改变位置
+    zh-Hant = 改變位置
+    ar = تغيير الموقع
+    th = เปลี่ยนสถานที่ตั้ง
+    uk = Змініть розташування
+    pt = Mudar o local
+    es = Cambiar ubicación
+    ro = Schimbare locație
+    pl = Zmień lokalizację
+    sk = Zmeniť polohu
+    it = Cambia posizione
+    fr = Modifier l'emplacement
+    ja = 位置を変更してください
+    vi = Thay đổi địa điểm
 
   [message_invalid_feature_position]
-    en = A place cannot be situated in this location.
-    ru = Объект не может находиться в этом местоположении.
+    en = An object can't be located here
```

Comment:
тут надо placed

> On Mar 22, 2016, at 14:59, Alexander Marchuk <notifications@github.com> wrote:
> 
> In strings.txt <https://github.com/mapsme/omim/pull/2402#discussion_r56974416>:
> 
> >  
> >    [message_invalid_feature_position]
> > -    en = A place cannot be situated in this location.
> > -    ru = Объект не может находиться в этом местоположении.
> > +    en = An object can't be located here
> А "located here" разве не означает "найден здесь"?
> 
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub <https://github.com/mapsme/omim/pull/2402/files/7068d5ef06d0a7211a4b5265be7d866b4e5c4250#r56974416>